code/modules/admin/topic.dm

```diff
@@ -30,10 +30,10 @@
 
 		var/msg = "<font color='red' size='4'><b>- AdminHelp marked as IC issue! -</b></font><br>"
 		msg += "<font color='red'><b>Losing is part of the game!</b></font><br>"
-		msg += "<font color='red'>Your character will frequently die, sometimes without even a possibility of avoiding it. Events will often be out of your control. No matter how good or prepared you are, sometimes you just lose.</font>"
-
+		msg += "<font color='red'>Your character will frequently die, sometimes without even a possibility of avoiding it. Events will often be out of your control. No matter how good or prepared you are, sometimes you just lose.</font><br>"
+		msg += "<font color='red'>We have opened the admin complaints subforum for you to save you some time in creating your ban request.</font>"
 		C << msg
-
+		C << link("https://tgstation13.org/phpBB/viewforum.php?f=23")

```

Comment:
have it automatically select the first guy who namechecks if you're going to go all the way.